### PR TITLE
Travis ci build and test

### PR DESCRIPTION
**Pull request by arl**
All is built using `bootstrap.py`, client + server

- [x] client tests (somebody knows the command?)
- [x] server tests

### CODE REVIEW

File: bootstrap.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import stat
 import subprocess as sp
 import sys
+import argparse
```
Comment (voidexp) [inline bootstrap.py]: sort imports

File: bootstrap.py [hunk 2]
```diff
@@ -177,11 +182,19 @@ def venv_setup(venv_path, inst_path):
 def server_install(root_path):
     env = dict(os.environ)
     env['GOPATH'] = os.path.abspath(root_path)
+    print('Installing server')
```
Comment (voidexp) [inline bootstrap.py]: move this to line 219, for consistency

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+sudo: false # enable container-based infrastructure
+
+addons:
+  apt:
+    sources:
+      - deadsnakes # source required so it finds the package definition below
+    packages:
+      # packahe list:
+      # https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise
+      - python3.5 python3.5-dev python3.5-venv check libglew-dev
```
Comment (voidexp) [inline .travis.yml]: If we're not going to invoke tests on dependencies (that should be done by their own CI), `check` is unneded.

File: bootstrap.py (hunk 1 above)
Comment (arl) [reply bootstrap.py]: done

File: bootstrap.py (hunk 2 above)
Comment (arl) [reply bootstrap.py]: done to 232

File: .travis.yml (hunk 3 above)
Comment (arl) [reply .travis.yml]: but without it, `bootstrap` complains. Maybe there's an option to not do dependencies test. But i dont see why we shouldn't do them. As long as deps don't have their own continuous integration tests, we should at least do them here

**Pull request merged**